src/Scaffolding/VS.Web.CG.Mvc/Identity/IdentityGeneratorTemplateModel2.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 namespace Microsoft.VisualStudio.Web.CodeGenerators.Mvc.Identity
```

Comment:
nit: is it worth putting a comment in here for what the distinction is between IdentityGeneratorTemplateModel2 and IdentityGeneratorTemplateModel does this map to a specific version i.e. pre 6 or something?

Reply:
We actually always use IdentityGeneratorTemplateModel2 as seen [here](https://github.com/dotnet/Scaffolding/blob/82f93025d6f1dd2f20dcac5319e722407e165647/src/Scaffolding/VS.Web.CG.Mvc/Identity/IdentityGeneratorTemplateModelBuilder.cs#L192). 
I didn't write this but I imagine it for extensibility for other developers to add a different template model and use IdentityGeneratorTemplateModel as a base class.